src/core/synchronization/autosync.rs

```diff
@@ -68,6 +70,7 @@ impl AutosyncProcess {
         estimator.every_nth_frame.store(every_nth_frame.max(1) as usize, SeqCst);
 
         let mut comp_params = ComputeParams::from_manager(stab);
+        stab.gyro.read().copy_org_into(&mut comp_params.gyro);
         comp_params.gyro.raw_imu = stab.gyro.read().raw_imu.clone();

```

Comment:
Instead of this and the entire `copy_org_into`, I would add a parameter to `from_manager` named `full_gyro` and used `.clone()` instead of `.clone_quaternions()` if `full_gyro` is true